translations/messages/nl-be.yaml

```diff
@@ -31,6 +33,7 @@ messages:
     highestversionAvailable: "De laatste versie is v{version}"
     formIsValid: "Je formulier is correct"
     formIsInvalid: "Je formulier is niet correct. Vul al de velden in."
+    previewNotUpdatedBecauseOfErrorsInTtl: "Preview niet geüpdate door errors in de ttl code"
```

Comment:
Almost, it's really "geüpdatet"
https://www.vlaanderen.be/team-taaladvies/taaladviezen/geupdatet-geupdatete-geupdate-geupdated